src/moscot/backends/ott/solver.py

```diff
@@ -97,6 +98,14 @@ def _create_geometry(
             if is_linear_term:
                 self._graph_in_linear_term = True
```

Comment:
> I think we should now check only for graph in linear term AND cost matrix is not instantiated

I think this is a sound option but I am not sure I understand how a geometry can both be graph and be a cost_matrix. since `x.is_cost_matrix` is handled above.

Reply:
The reason why we need the `_graph_in_linear_term` is to indicate whether we need an `OTTOutput` or a `GraphOTTOutput`. If we instantiate the geodesic cost matrix, we need the `OTTOutput`, hence, internally, this is handled as if we had a cost matrix.